File: src/main/kotlin/xingkong/loopicserver/picproc/AddTagByDir.java
package xingkong.loopicserver.picproc;


import xingkong.loopicserver.picproc.tagImg.TagManager;

import java.util.Scanner;

public class AddTagByDir {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        TagManager tm = new TagManager();
        System.out.println("---------------------------------");
        String dir = in.next();
        tm.addTagByDir(dir);

    }
}

File: src/main/kotlin/xingkong/loopicserver/picproc/AddTagByDir.kt
package xingkong.loopicserver.picproc


import xingkong.loopicserver.picproc.tagImg.TagManager
import java.util.*

object AddTagByDir {
    @JvmStatic
    fun main(args: Array<String>) {
        val `in` = Scanner(System.`in`)
        val tm = TagManager()
        println("---------------------------------")
        val dir = `in`.next()
        tm.addTagByDir(dir)

    }
}
